core/storage/exploration/gae_models_test.py

```diff
@@ -894,3 +895,125 @@ def test_get_at_least_editable(self) -> None:
+            'most_recently_reached_checkpoint_exp_version':
+                base_models.EXPORT_POLICY.NOT_APPLICABLE,
+            'most_recently_reached_checkpoint_state_name':
+                base_models.EXPORT_POLICY.NOT_APPLICABLE,
+        }
+        fetched_dict = (
+            exp_models.TransientCheckpointUrlModel.get_export_policy())
+        self.assertEqual(
+            expected_dict['exploration_id'],
+            fetched_dict['exploration_id'])
+        self.assertEqual(
+            expected_dict['furthest_reached_checkpoint_exp_version'],
+            fetched_dict['furthest_reached_checkpoint_exp_version'])
+        self.assertEqual(
+            expected_dict['furthest_reached_checkpoint_state_name'],
+            fetched_dict['furthest_reached_checkpoint_state_name'])
+        self.assertEqual(
+            expected_dict['most_recently_reached_checkpoint_exp_version'],
+            fetched_dict['most_recently_reached_checkpoint_exp_version'])
+        self.assertEqual(
+            expected_dict['most_recently_reached_checkpoint_state_name'],
+            fetched_dict['most_recently_reached_checkpoint_state_name'])
+
+    def test_create_new_object(self) -> None:
+        exp_models.TransientCheckpointUrlModel.create(
+            'exp_id', 'progress_id')
+        transient_checkpoint_url_model = (
+            exp_models.TransientCheckpointUrlModel.get_by_id(
+                'progress_id'))
+
+        # Ruling out the possibility of None for mypy type checking.
+        assert transient_checkpoint_url_model is not None
+        self.assertEqual(
+            transient_checkpoint_url_model.exploration_id,
+            'exp_id')
+        self.assertIsNone(
+            transient_checkpoint_url_model.
+            most_recently_reached_checkpoint_exp_version)
+        self.assertIsNone(
+            transient_checkpoint_url_model.
+            most_recently_reached_checkpoint_state_name)
+        self.assertIsNone(
+            transient_checkpoint_url_model.
+            furthest_reached_checkpoint_exp_version)
+        self.assertIsNone(
+            transient_checkpoint_url_model.
+            furthest_reached_checkpoint_state_name)
+
+    def test_get_object(self) -> None:
+        exp_models.TransientCheckpointUrlModel.create(
+            'exp_id', 'progress_id')
+        expected_model = exp_models.TransientCheckpointUrlModel(
+            exploration_id='exp_id',
+            most_recently_reached_checkpoint_exp_version=None,
+            most_recently_reached_checkpoint_state_name=None,
+            furthest_reached_checkpoint_exp_version=None,
+            furthest_reached_checkpoint_state_name=None
+        )
+
+        actual_model = (
+            exp_models.TransientCheckpointUrlModel.get_by_id(
+                'progress_id'))
+
+        # Ruling out the possibility of None for mypy type checking.
+        assert expected_model is not None
+        assert actual_model is not None
```

Comment:
Instead of doing this you can use `get` with `strict=True` and it doesn't need to be checked by the `assert`.